File: TransCoder_tokenized_test_set_functions/python/LONGEST_CONSECUTIVE_SUBSEQUENCE.py
def findLongestConseqSubseq ( arr, n ) :
    s = Set ( )
    ans = 0
    for ele in arr :
        s . add ( ele )
    for i in range ( n ) :
        if ( arr [ i ] - 1 ) not in s :
            j = arr [ i ]
            while ( j in s ) :
                j += 1
            ans = max ( ans, j - arr [ i ] )
    return ans

File: results/py/LONGEST_CONSECUTIVE_SUBSEQUENCE.cpp
int f_filled ( int arr [ ] , int n ) {
  set < int > s ;
  int ans = 0 ;
  for ( int ele : arr ) s . insert ( ele ) ;
  for ( int i = 0 ;
  i < n ;
  i ++ ) {
    if ( ( arr [ i ] - 1 ) != s . count ( arr [ i ] ) ) {
      int j = arr [ i ] ;
      while ( ( j < s . size ( ) ) && ( s [ j ] == arr [ i ] ) ) j ++ ;
      ans = max ( ans , j - arr [ i ] ) ;
    }
  }
  return ans ;
}
